docs/DeveloperGuide.md

```diff
@@ -277,12 +278,37 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+| `* *`    | new home tutor                                      | track the performance of my students   | prove to their parents that they are improving and my effectiveness as a tutor                |
+| `* * *`     | new home tutor | track all my appointments          | I don’t forget or double book myself                                               |
+| `* *`      | new home tutor | keep track of my finances–such as which client has paid me for my work over a month | reduce my stress when it comes to keeping track of who has paid me for my work, and so forth. |
+| `* *`      | busy home tutor | seamlessly create new tasks | quickly and efficiently set up my daily routine |
+| `* *` | busy home tutor | be reminded about my upcoming appointments for the day | don't accidentally forget |
+| `* *` | returning home tutor | re-implementation of more information in a large amount to be easy | won't have to stress on adding information en masse |
+| `* *` | home tutor with clients who are nearing exam season | properly ensure that they are improving as planned, and also allow them to set benchmarks that I can remember | |
+| `* *` | busy home tutor | block certain times out for lunch and dinner | won't accidentally overwrite those times with an additional client |
+| `* * *` | experienced home tutor | delete students I am no longer teaching on TutorRec | they don’t clutter my interface |
+| `* * *` | experienced home tutor | update student information and details | ensure accurate records are maintained |
+| `* * *` | passionate home tutor | TutorRec to keep notes for each student | tailor my teaching style accordingly |
+| `* *` | experienced home tutor with many students | ability to categorise students by skill level, subject or group (p1, p2, p3…) | quickly locate their information when needed |
+| `* *` | experienced home tutor | flexibility to choose what to teach my students (custom lessons) | personalise each lesson for different students |
+| `* *` | experienced home tutor | easily identify students with weak performance | focus on weaker students |
+| `* *` | experienced home tutor | view a student’s records for the length of time I have tutored him/her | |
+| `* *` | experienced home tutor | get some insights and analytics on a student’s performance over time | further refine my teaching methods accordingly |
+| `* *` | busy home tutor | quickly reschedule my appointments with my clients | fit my ever-changing schedule, preventing a large amount of hassle |
+| `* *` | tutor who just received a new wave of clients | separate my old and new clients | organization remains orderly and easy-to-follow |
+| `* * *` | busy home tutor | make quick notes about my students | won't lose track of information specific to each client |
+| `* *` | assignment-ridden home tutor | note cancellations in my schedule due to increased workload from my end | my schedule is accurate to reality |
```

Comment:
Maybe add the word 'ensure' in front of 'my schedule is...' to keep in line with header grammar?

Reply:
Not sure what you mean here. I think the grammar seems fine?